set.mm

```diff
@@ -78255,11 +78289,19 @@ Axiom of Replacement (unlike ~ resfunexg ).  (Contributed by NM,
   $}
 
   ${
-    $d A f $.  $d B f $.
+    $d A f $.  $d B f $.  $d C f $.  $d D f $.
```

Comment:
The OLD proof depends on fewer dv conditions.  I understand that f is bounded, and that a change of variable like 
{ f | ( f : A -1-1-onto-> B /\ ph ) } = { g | ( g : A -1-1-onto-> B /\ [ g / f ] ph ) }
holds, but that is hardly provable without ax-12.

Note that the old proof does not depend on ax-10 .. ax-12.
I think this is not a valid shortening.

Reply:
The new proofs neither depend on ax-10, ax-11, ax-12. 
For all of the four theorems fabexg, fabexgOLD, f1oabexg, f1oabexgOLD, calling `sh tr xxx /ax/match ax-*`  results in 
`ax-mp ax-1 ax-2 ax-3 ax-gen ax-4 ax-5 ax-6 ax-7 ax-8 ax-9 ax-ext ax-sep ax-nul ax-pow ax-pr ax-un`

Reply:
It is not the axiom list I am not satisfied with.

The expression A e. C is a convenient shorthand for E. x x = A, or by isset for A e. V_.  So a fix would simply use A e. V_ and B e. V_ in the proof first, and then in a final step relax these expressions to A e. C and B e. D by elisset.

C and D are not subject to any restriction to fulfill their role. Here you artificially add some.  Get rid of them.